### PR TITLE
Grekhaus more sponges

### PR DESCRIPTION
**Pull request by Grekhaus**
#### Summary
```SUMMARY: Balance "Makes sponges less rare and craftable"```

#### Purpose of change
Sponges are currently very rare and hard to find, as they were not added to very many spawn locations when initially implemented.

#### Describe the solution
This PR adds sponges to more locations, including kitchens, bathrooms, hospitals, labs and the general tool pool. It also adds a new recipe to craft sponges, auto-learned at fabrication 3.

#### Describe alternatives you've considered
-Allowing fibrous stalks to be used as sponges.
-Adding sponges (ie the loffa plant) to the forage pool.
-Requiring sewing instead of/in addition to chemical making for the sponge recipe

### CODE REVIEW

File: data/json/itemgroups/item_groups.json [hunk 1]
```diff
@@ -5305,6 +5307,7 @@
       [ "bleach", 20 ],
       [ "bandages", 50 ],
       [ "cotton_ball", 50 ],
+      [ "sponge", 20 ],
```
Comment (Fris0uman) [inline data/json/itemgroups/item_groups.json]: Are sponges really part of dissection equipment?

File: data/json/recipes/other/tool.json [hunk 2]
```diff
@@ -374,6 +374,18 @@
     "autolearn": true,
     "components": [ [ [ "colander_steel", 1 ], [ "wire_mesh", 1 ], [ "splatter_guard", 1 ] ] ]
   },
+  {
+    "type": "recipe",
+    "result": "sponge",
```
Comment (Fris0uman) [inline data/json/recipes/other/tool.json]: I have some doubt that you can make a sponge by putting a rag, some plastic and dry grass into a cooking pot.
If you want to add a recipe for a cleaning tool you should probably find one that is actually craftable and add it to the game with it's recipe.

File: data/json/itemgroups/item_groups.json (hunk 1 above)
Comment (Grekhaus) [reply data/json/itemgroups/item_groups.json]: They're not used for the dissection itself, but for the post-dissection cleanup, as with the bleach and cotton balls.

**Comment by Grekhaus** [comment]
Going to put more thought into the recipe and then figure out how to update this branch to the new build.

**Comment by Fris0uman** [comment]
The changes to item list seemed fine to me and could be enough to solve the sponge issue. So you could make that into a PR for a start while you figure out how to make a craftable cleaning tool.

**Comment by anothersimulacrum** [comment]
You may want to check out [this forum thread](https://discourse.cataclysmdda.org/t/sponge-being-non-craftable-is-intended/21331/8)

**Comment by Fris0uman** [comment]
This #34124 should take care of the wash cloth, so if you're still intrested you can just submitt your itemgroup PR and everything should be fixed.

**Pull request closed without merging**